src/shared/App.js

```diff
@@ -73,6 +66,9 @@ function App() {
                     <ProtectedRoute exact path="/team">
                       Team page
                     </ProtectedRoute>
+                    <ProtectedRoute exact path="/store">
+                      <Store />
+                    </ProtectedRoute>
```

Comment:
for now let's put this as a non-protected route, i.e. at the same level as `/chakraExpo`

Reply:
Done!